zooma-mappings-wf/Snakefile

```diff
@@ -0,0 +1,376 @@
+import re
+from os import listdir
+from datetime import datetime
+from snakemake.utils import min_version
+
+min_version("6.6.1")
+
+# This workflow updates all ontology mappings in condensed SDRF files, and then
+# updates all experiment designs on ves-hx-76:8080
+
+# The minimal required inputs are configuration files in xml format
+
+include: "rules/common.smk"  # python helper functions 
+
+
+# below probably no longer needed bec of regex in common.smk
+wildcard_constraints:
+    accession = 'E-\D{1,20}-\d{1,20}'
+
+
+localrules: check_zooma, remove_aux_files
+
+rule all:
+    input:
+        completed=f"{working_dir}/logs_zooma_mappings/copy_reports_to_targetDir.log"
+
+
+rule check_zooma:
+    """
+    Check that zooma returns successful http code
+    """
+    conda: 
+        "envs/base.yaml"
+    resources:
+        load=config['load_zooma_jobs']
+    log:
+        "{working_dir}/{accession}/logs_zooma_mappings/{accession}-check_zooma.log"
+    output:
+        temp("{working_dir}/{accession}/check_zooma.done")
+    params:
+        zoomaMetadataUrl=config['zoomaMetadataUrl']
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
```

Comment:
On other workflows I keep creating the directory for the logs, but maybe these are created on its own by snakemake based on this?

Reply:
Yes, it is created by Snakeme if you specify a log file with `log:` in the rule